app/controllers/deep_links_controller.rb

```diff
@@ -0,0 +1,21 @@
+class DeepLinksController < ApplicationController
+  def mobile; end
+
+  # Apple Application Site Association
+  def aasa
+    supported_apps = ["R9SWHSQNV8.com.forem.app"]
```

Comment:
I'm not overly familiar with deep-linking, is hard-coding this here a good idea? Should it maybe be some config value instead? Or an env var or something?

Reply:
Hardcoding this here allows for the Forem app to be supported by default. Actually all forems have the DEV app hardcoded right now ([https://community.codenewbie.org/apple-app-site-association](https://community.codenewbie.org/apple-app-site-association) for example). Now that's definitely not ideal, because the DEV app only supports DEV compared to the Forem app that will support any Forem.

Ideally I'm hoping to continue to expand on exposing configurations creators will be able to leverage in order to have control over this. Adding a selector to `/admin/consumer_apps` where they can choose which one of their consumer apps will be the one users will be deep linked into from the banner shouldn't be too difficult, but felt like I wanted to test this out with the TestFlight builds before going too deep in the weeds on configurability. Check out the [**feature 3** section of the RFC](https://github.com/forem/rfcs/blob/d28ba6ce61c2cda9d4018b4237ec4866be4359b9/text/0072-runtime-banner-with-deep-linking.md#admin-panel-configuration-feature-3) for more details.

tl;dr: It's not bad (specially compared to the current state we're on) and adding some configuration to make this more dynamic is indeed going to happen (soon 🤞 )

Reply:
I agree with @citizen428 that we should avoid hardcoding this and the below `"R9SWHSQNV8.to.dev.ios"` string -- but I'm also okay to revisit this in a separate, follow-up PR and extract it out if that seems easier (we could leave a `TODO: ` here for that if that would be helpful!)

Reply:
> we could leave a TODO: here for that if that would be helpful!

I think it would, and I just did 😃

Reply:
Thanks for the context @fdoxyz, a comment is good enough for now.